### PR TITLE
[wix-style-react] - <CounterBadge/> - Add light skin and shadow prop

### PR DESCRIPTION
**Pull request by mjurfest**
### 🔦 Summary

- Add `light` skin
- Add `showShadow` prop

### ✅ Checklist

- [X] 👨‍🎨 UX change is approved by the Design System UX  @milkyfruit 
- 📚 Change is documented
  - [X] Story
  - [X] API description

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6614

File: packages/wix-style-react/src/CounterBadge/docs/examples.js [hunk 1]
```diff
@@ -74,6 +78,9 @@ export const custom = `
   <Cell span={1}>
     <CounterBadge skin="success">New!</CounterBadge>
   </Cell>
+  <Cell span={1}>
+    <CounterBadge skin="light" showShadow>Shadow</CounterBadge>
+  </Cell>
```
Comment (EgoziE) [inline packages/wix-style-react/src/CounterBadge/docs/examples.js:83]: Should be extracted to a separate example

**Review by EgoziE — CHANGES_REQUESTED** [review]

**Comment by EgoziE** [comment]
@milkyfruit Ready for UX review
Try the shadow prop with all skins, something like:
```
<Layout>
  <Cell>
    <CounterBadge skin="general" showShadow>
      1
    </CounterBadge>
  </Cell>
  <Cell>
    <CounterBadge skin="standard" showShadow>
      1
    </CounterBadge>
  </Cell>
  <Cell>
    <CounterBadge skin="neutralStandard" showShadow>
      1
    </CounterBadge>
  </Cell>
  <Cell>
    <CounterBadge skin="danger" showShadow>
      1
    </CounterBadge>
  </Cell>
  <Cell>
    <CounterBadge skin="warning" showShadow>
      1
    </CounterBadge>
  </Cell>
  <Cell>
    <CounterBadge skin="urgent" showShadow>
      1
    </CounterBadge>
  </Cell>
  <Cell>
    <CounterBadge skin="success" showShadow>
      1
    </CounterBadge>
  </Cell>
  <Cell>
    <CounterBadge skin="light" showShadow>
      1
    </CounterBadge>
  </Cell>
</Layout>
```

**Review by milkyfruit — APPROVED** [review]
@EgoziE @mjurfest from a design side, both props look good.

**Review by EgoziE — APPROVED** [review]

**Pull request merged**